src/components/manage/Blocks/DataFigure/ImageSidebar.jsx

```diff
@@ -112,15 +124,15 @@ const ImageSidebar = ({
                 iconAction={
                   data.url
                     ? () => {
-                        resetSubmitUrl();
-                        onChangeBlock(block, {
-                          ...data,
-                          url: '',
-                        });
-                      }
+                      resetSubmitUrl();
+                      onChangeBlock(block, {
+                        ...data,
+                        url: '',
+                      });
+                    }
                     : () => openObjectBrowser()
                 }
-                onChange={() => {}}
+                onChange={() => { }}
```

Comment:
@avoinea Please ignore these. Don't know why my linter is formatting onSave, have to reconfigure it ;/